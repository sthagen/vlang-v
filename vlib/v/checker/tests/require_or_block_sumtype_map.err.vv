type Abc = string | int

fn main() {
	x := map[string]Abc{}
	_ := x['nonexisting']
}
