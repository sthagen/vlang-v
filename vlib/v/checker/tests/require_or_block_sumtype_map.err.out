vlib/v/checker/tests/require_or_block_sumtype_map.err.vv:5:8: error: `or {}` block required when indexing a map with sum type value
    3 | fn main() {
    4 |     x := map[string]Abc{}
    5 |     _ := x['nonexisting']
      |           ~~~~~~~~~~~~~~~
    6 | }
